src/set_theory/ordinal_arithmetic.lean

```diff
@@ -1984,8 +1984,7 @@ theorem CNF_ne_zero {b o : ordinal} (b0 : b ≠ 0) (o0 : o ≠ 0) :
   CNF b o = (log b o, o / b ^ log b o) :: CNF b (o % b ^ log b o) :=
 by unfold CNF; rw [dif_neg b0, dif_neg b0, CNF_rec_ne_zero b0 o0]
 
-@[simp] theorem one_CNF {o : ordinal} (o0 : o ≠ 0) :
-  CNF 1 o = [(0, o)] :=
+@[simp] theorem one_CNF {o : ordinal} (o0 : o ≠ 0) : CNF 1 o = [(0, o)] :=
```

Comment:
This is going to conflict with #12733

Reply:
I'd rather get this one out of the way first. I'll mark the other one as depending on this.